### PR TITLE
options rendering page translated

### PR DESCRIPTION
**Pull request by Slrhoseini**

### CODE REVIEW

**Comment by mostafa-nematpour** [comment]
Thank you for your contribution.
First, please remove the `package-lock.json` from the PR.

**Comment by Slrhoseini** [comment]
package-lock.json has been removed.

File: src/api/options-rendering.md [hunk 1]
```diff
@@ -78,19 +78,19 @@ Configure runtime compiler options for the component's template.
...
+## اسلات‌ها<sup class="vt-badge ts"/> {#slots}
 
-An option to assist with type inference when using slots programmatically in render functions. Only supported in 3.3+.
+یک گزینه برای کمک به استخراج نوع استنتاج هنگام استفاده برنامه‌نویسی از اسلات‌ها در توابع رندر. تنها در نسخه 3.3 به بالا پشتیبانی میشود.
 
-- **Details**
+- **جزئیات**
 
-  This option's runtime value is not used. The actual types should be declared via type casting using the `SlotsType` type helper:
+  مقدار ران‌تایم این گزینه استفاده نمیشود. تایپ‌های واقعی باید از طریق تعیین تایپ با استفاده از کمک‌کننده تایپ `SlotsType` اعلام شوند.
```
Comment (mostafa-nematpour) [inline src/api/options-rendering.md:93]:
```suggestion
  مقدار ران‌تایم این آپشن استفاده نمی‌شود. تایپ‌های واقعی باید از طریق تعیین تایپ با استفاده از کمک‌کننده تایپ `SlotsType` اعلام شوند.
```

**Review by mostafa-nematpour — COMMENTED** [review]
Review the changes and accept them.
They help improve the translation quality.

**Comment by Slrhoseini** [comment]
all changes has been accepted.

**Comment by mostafa-nematpour** [comment]
Thank you

**Pull request merged**